docs/how-to-guides/backup-rook-ceph-volumes.md

```diff
@@ -0,0 +1,267 @@
+# Backup Rook Ceph Volume on S3 using Velero
+
+## Contents
+
+- [Introduction](#introduction)
+- [Learning objectives](#learning-objectives)
+- [Prerequisites](#prerequisites)
+- [Steps](#steps)
+  - [Step 1: Deploy Velero](#step-1-deploy-velero)
+    - [Config](#config)
+    - [Deploy](#deploy)
+  - [Step 2: Deploy sample workload](#step-2-deploy-sample-workload)
+  - [Step 3: Annotate pods](#step-3-annotate-pods)
+  - [Step 4: Backup entire namespace](#step-4-backup-entire-namespace)
+  - [Step 5: Restore Volumes](#step-5-restore-volumes)
+    - [Same Cluster](#same-cluster)
+    - [Different Cluster](#different-cluster)
+    - [Restore](#restore)
+- [Additional resources](#additional-resources)
+
+## Introduction
+
+[Rook](https://rook.io/) is a component of Lokomotive which provides storage on Packet cloud. Taking
+regular backup of the data to a remote server is an essential strategy for disaster recovery.
+
+[Velero](https://velero.io/) is another component of Lokomotive which helps you to backup entire
+namespaces, including volume data in them.
+
+## Learning objectives
+
+This document will walk you through the process of backing up a namespace including the volume in
+it.
+
+## Prerequisites
+
+- A Lokomotive cluster deployed on a Packet cloud and accessible via `kubectl`.
+
+- Rook Ceph installed by following [this guide](./rook-ceph-storage.md).
+
+- `aws` CLI tool [installed](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html).
+
+- S3 bucket created by following [these
+  instructions](https://github.com/vmware-tanzu/velero-plugin-for-aws/blob/8d31a11/README.md#create-s3-bucket).
+
+- Velero user in AWS created by following [these
+  instructions](https://github.com/vmware-tanzu/velero-plugin-for-aws/blob/8d31a11/README.md#option-1-set-permissions-with-an-iam-user).
+
+- Velero CLI tool [downloaded](https://github.com/vmware-tanzu/velero/releases/tag/v1.4.2) and
+  installed in the `PATH`.
+
+## Steps
+
+### Step 1: Deploy Velero
+
+#### Config
+
+Create a file `velero.lokocfg` with the following contents:
+
+```tf
+variable "velero_aws_credentials" {}
+
+component "velero" {
+  restic {
+    credentials = var.velero_aws_credentials
```

Comment:
```suggestion
    credentials =file("./credentials-velero")
```

Better way would be to use the file function as users might be skeptical of adding AWS secret keys again in another file.